src/tester.rs

```diff
@@ -85,6 +85,14 @@ impl<'a> LangTester<'a> {
         self
     }
 
+    /// Specify the number of simultaneous running test cases. By default, Rust
```

Comment:
Maybe change the second sentence to "Defaults to the number of available CPUs."?

Reply:
Fixed: 8c6b2b1280dc51dbfeb7aa0da7f7fa90794d0c41